lib/DI/ContainerConfigurator.php

```diff
@@ -82,6 +82,7 @@ function configure(ContainerBuilder $container) {
     $container->autowire(\MailPoet\Router\Endpoints\Track::class)->setPublic(true);
     $container->autowire(\MailPoet\Statistics\Track\Clicks::class);
     $container->autowire(\MailPoet\Statistics\Track\Opens::class);
+    $container->autowire(\MailPoet\Statistics\Track\WooCommerceOrders::class)->setPublic(true);
```

Comment:
It is injected into `Hooks.php`, isn't it? I thought @costasovo said we should do all services public at the moment.

Reply:
It is injected, but that doesn't mean it must be public. We need public services only those that are directly accessed on the container. I am not sure why @costasovo would say that but Symfony claims it is a bad practice: https://symfony.com/blog/new-in-symfony-3-4-services-are-private-by-default

Reply:
Hey, you are correct about public services. I've created all public at the beginning since container was injected in Initializer and we indeed were getting many services directly. We should add new ones as a private and refactor other to private when it is possible.